src/main/java/com/gluonhq/attach/AttachArtifactResolver.java

```diff
@@ -72,6 +75,7 @@
                     }
                     return resolve.stream();
                 })
+                .filter(Objects::nonNull)
```

Comment:
The `null` values are already filtered out at line 70 and the `flatMap` section doesn't seem to introduce new possible `null` values? However, I see that `MavenArtifactResolver.resolve()` can return `null` in some cases. Isn't it better to let it return an empty Set instead for those cases?

Reply:
Yes, `MavenArtifactResolver` might return null (for instance looking for util-ios.jar with an old Attach version). That produces an `ArtifactNotFoundException` exception (from a try-catch inside `resolve`).
We return an empty stream, so the filter is indeed unnecessary. I've removed it.